djstripe/stripe_objects.py

```diff
@@ -832,14 +855,18 @@ class Meta:
     webhook_message = StripeJSONField(stripe_name="data", help_text="data received at webhook. data should be considered to be garbage until validity check is run and valid flag is set")
 
     def str_parts(self):
+        """Return a nice string repr of this data object."""
         return [
             "type={type}".format(type=self.type),
         ] + super(StripeEvent, self).str_parts()
 
     def api_retrieve(self, api_key=settings.STRIPE_SECRET_KEY):
-        # OVERRIDING the parent version of this function
-        # Event retrieve is special. For Event we don't retrieve using djstripe's API version. We always retrieve
-        # using the API version that was used to send the Event (which depends on the Stripe account holders settings
+        """
```

Comment:
We don't want to overwrite the docstring here. The functionality is the same; the comments are just there to let devs know what's going on.